<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoDialog</class>
 <widget class="QDialog" name="InfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Info / Edit</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tab_widget">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="usesScrollButtons">
          <bool>false</bool>
         </property>
         <property name="tabsClosable">
          <bool>false</bool>
         </property>
         <widget class="QWidget" name="tab_1">
          <attribute name="icon">
           <iconset>
            <normaloff>:/Icons/info.svg.png</normaloff>:/Icons/info.svg.png</iconset>
          </attribute>
          <attribute name="title">
           <string notr="true">Info</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="horizontalSpacing">
            <number>20</number>
           </property>
           <item row="6" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btn_close1">
               <property name="text">
                <string notr="true">Close</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="4" column="1">
            <widget class="QWidget" name="ui_info_widget" native="true">
             <layout class="QGridLayout" name="gridLayout_4">
              <property name="horizontalSpacing">
               <number>3</number>
              </property>
              <item row="1" column="0" colspan="5">
               <widget class="Line" name="line">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QTableWidget" name="table_info">
                <attribute name="horizontalHeaderVisible">
                 <bool>false</bool>
                </attribute>
                <attribute name="horizontalHeaderStretchLastSection">
                 <bool>true</bool>
                </attribute>
                <attribute name="verticalHeaderVisible">
                 <bool>false</bool>
                </attribute>
               </widget>
              </item>
              <item row="0" column="0" colspan="4">
               <widget class="QLabel" name="lab_title">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string notr="true">Title</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="lab_subheader">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string notr="true">by Artist</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0" colspan="5">
               <widget class="QListWidget" name="list_paths">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>1</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::MinimumExpanding</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>5</width>
                  <height>350</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="5" column="2" colspan="2">
               <layout class="QVBoxLayout" name="verticalLayout">
                <property name="spacing">
                 <number>2</number>
                </property>
                <item>
                 <widget class="Gui::CoverButton" name="btn_image">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="focusPolicy">
                   <enum>Qt::StrongFocus</enum>
                  </property>
                  <property name="text">
                   <string notr="true"/>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="Line" name="line_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Preferred</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>320</width>
                    <height>5</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="btn_write_cover_to_tracks">
                  <property name="text">
                   <string notr="true">Write cover into tracks...</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="icon">
           <iconset>
            <normaloff>:/Icons/lyrics.svg.png</normaloff>:/Icons/lyrics.svg.png</iconset>
          </attribute>
          <attribute name="title">
           <string notr="true">Lyrics</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3"/>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="icon">
           <iconset>
            <normaloff>:/Icons/edit.svg.png</normaloff>:/Icons/edit.svg.png</iconset>
          </attribute>
          <attribute name="title">
           <string notr="true">Edit</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_8"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Loading files...</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QProgressBar" name="progressBar">
         <property name="maximum">
          <number>0</number>
         </property>
         <property name="value">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::CoverButton</class>
   <extends>QPushButton</extends>
   <header>Gui/Covers/CoverButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resources/Icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_close1</sender>
   <signal>clicked()</signal>
   <receiver>InfoDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>592</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_close1</sender>
   <signal>clicked()</signal>
   <receiver>InfoDialog</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>592</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
